packages/nx/src/config/workspaces.ts

```diff
@@ -431,6 +432,15 @@ function assertValidWorkspaceConfiguration(
   }
 }
 
+function isNrwlAngularInstalled() {
+  try {
+    require.resolve('@nrwl/angular');
```

Comment:
Some debate on whether `require.resolve` vs reading package.json and checking deps is better for this check.

This **_could_** potentially have a false positive if there was @nrwl/angular found in one of the dependency chains of another installed library, but that should be relatively rare. In any case, the behavior exhibited would be the currently existing behavior, so its a low impact.